src/constants/index.js

```diff
@@ -13,6 +13,7 @@ import * as OSType from './OSType'
 import * as Overlay from './Overlay'
 import * as PageContentFormat from './PageContentFormat'
 import * as PlatformType from './PlatformType'
+import * as Role from './Role'
```

Comment:
Don't forget to export this at the bottom

Reply:
moved it to the constants folder in role module